frontend/device/kindle/device.lua

```diff
@@ -1046,6 +1074,20 @@ local KindleScribe = Kindle:extend{
     canDoSwipeAnimation = yes,
 }
 
+local KindleColorSoft = Kindle:extend{
+    model = "KindleColorSoft", -- Signature Edition, has light sensor
```

Comment:
nit: copy/pasted comment should be removed.

Reply:
I kept it as its a signature edition as well

Reply:
Maybe worth explicitly saying "Kindle Colorsofts all have ambient light sensor" because there are no non-signature colorsofts? Just for clarity (and future)

Reply:
But it already says so right below in `hasLightSensor = yes`? Is there a particular point to the comment?